modules/ROOT/pages/http/http-post-form-task.adoc

```diff
@@ -66,7 +58,7 @@ For example:
     </flow>
 ----
 
-. After the HTTP Connector, add a Choice scope with two alternative paths. Set the first to be executed when the following MEL expression is true:
+. After the HTTP Connector, add a `choice` router with two alternative paths. Set the first path to execute when the following MEL expression is true:
```

Comment:
```suggestion
. In the flow, after the HTTP Connector add a `choice` router with two alternative paths, in which the first path executes when the following MEL expression is true:
```
Should we change earlier references to Choice router to "`choice` router"?
Also, is the expression literally `true` or does is just have the condition of being true?

Reply:
If you go on to document the second alternative, then I suggest substeps:
```suggestion
. After the HTTP Connector, add a `choice` router with two alternative paths:
.. Set the first path to execute when the following MEL expression is true:
```